src/index.ts

```diff
@@ -0,0 +1,15 @@
+export class Challenge {
+    buzz(num: number) : Promise<any> {
+        return new Promise<any>((resolve, reject) => {
+            if (num % 3 === 0 && num % 5 === 0) {
```

Comment:
Please no space between if and (, 🙈
Single line `if` should not have any brackets